examples/webgl_materials_envmaps_hdr.html

```diff
@@ -36,12 +36,49 @@
 			var container, stats;
 			var camera, scene, renderer, controls;
 			var torusMesh, planeMesh;
-			var ldrCubeRenderTarget, hdrCubeRenderTarget, rgbmCubeRenderTarget;
+			var generatedCubeRenderTarget, ldrCubeRenderTarget, hdrCubeRenderTarget, rgbmCubeRenderTarget;
 			var ldrCubeMap, hdrCubeMap, rgbmCubeMap;
 
 			init();
 			animate();
 
+			function getEnvScene() {
+
+				var envScene = new THREE.Scene();
+
+				var geometry = new THREE.BoxBufferGeometry();
+    			geometry.deleteAttribute('uv');
+    			var roomMaterial = new THREE.MeshStandardMaterial({metalness: 0, side: THREE.BackSide});
+				var room = new THREE.Mesh(geometry, roomMaterial);
+				room.scale.setScalar(10);
+				envScene.add(room);
+
+				var mainLight = new THREE.PointLight(0xffffff, 0.2, 0, 2);
+				mainLight.position.set(0, 0, 0);
+				envScene.add(mainLight);
+
+				var lightMaterial = new THREE.MeshBasicMaterial();
+				lightMaterial.color.setScalar(20);
```

Comment:
We don't set colors to [ 20, 20, 20 ]. Color is the diffuse reflectance of the material, it is unit-less, and takes values in [ 0, 1 ].

Reply:
As far as I understand this is emulating a very bright light.
So there is no other way to do this than by going over 1?

Reply:
Setting the diffuse reflectance to a value greater than 1 means that more light is reflected than is incident. We should not be doing that.

You can add a light to the scene and increase the light's `intensity`.

You can also set `material.emissive` ( in [0, 1] ) and `material.emissiveIntensity`, which is unbounded.

Reply:
Hmm, `MeshBasicMaterial` doesn't have `emissive`...

So in order to create "RectAreaLights" when generating envMaps you recomend doing this:

```js
new THREE.MeshStandardMaterial( { emissive: 0xffffff, emissiveIntensity: 20 } );
```

Instead of this:

```js
new THREE.MeshBasicMaterial( { color: new THREE.Color( 0xffffff ).scale( 20 ) } );
```

Right?

Reply:
Actually, I wrote a `RectAreaLightHelper`, but I specifically set the rectangle color so it wouldn't over-bright or hue shift as the light intensity increases. I think you will have to experiment with it so you will understand what I am referring to.  :-)

Tip: `RectAreaLightHelper` must be added as a child of the light.

Reply:
I think what you are looking for is:
```js
new THREE.MeshLambertMaterial( { color: 0x000000, emissive: 0xffaaaa, emissiveIntensity: 20 } );
```

Reply:
You'll need tone-mapping and exposure control.

Reply:
Hmm... Even when tone-mapped, it is hue-shifting to white unless the exposure is very low... Bummer.

Well, that is a separate issue...

Reply:
Thanks for the discussion; should I change this to MeshLambertMaterial then?

Reply:
Or RectAreaLightHelper?

Reply:
Using `MeshLambertMaterial` sounds good to me.